server/tests/common.integration.js

```diff
@@ -2,6 +2,21 @@ import p from '../../package';
 import config from '../../config/config';
 
 const version = p.version.split('.').shift();
+
+let authToken;
+let auth2Token;
+let authTokenBad;
+
+if (config.jwtMode === 'rsa') {
+    authToken = config.testTokenRSA;
+    auth2Token = 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MTgzMSwidXNlcm5hbWUiOiJ1c2VyMiIsImVtYWlsIjoidXNlcjJAYW1pZGEuY29tIiwic2NvcGVzIjpbImFkbWluIl0sImlhdCI6MTUyNDA2OTgwMiwiZXhwIjoxNTI0MDczNDAyfQ.xFkgScAMLuwgUgzILrafbWYyOb3RcPmdnYkO5G79oo_a62Rc_j88WpknqDOXdQe4DBNX9roaEmy3Ehpvyvv9JXrg5OEnaGzPV2wE-Doc7QDkQ4lGr-YwG2YdYmh0_MXGUE66mzqViuX4BvGh7jtKvw9hsvjxm3pGTUNPSJ_6pHNuGr3wKC3cEtmQHTXITaCdam6HC3yuTlhT8HfH11ADUFdx8HUa-jpuZ7_YSqJ_GvIjSgQNt3UP4ErVvmDhXbes09RSM4Rw7pCoe0ljFwEywFP9wrwYLOskeaqjeUilARdVL2nSTgtKdIL2qBz3_bsbkf4r8SYedNEAo3UxbuyNPtNJltUK6HRGMW9NUkwWIfbf6dagHlh9bxKmrQYkSxy2zHa8nvcMaMbKvf0uoyfILfuyEFpv7HUVGForlZhtrQ-LsIpt2oBOFLc0CY3yVW2LGtAKgxZyWlgBsim49awGkRBSkLhuev528NNr1-mUeG0E3DzU5T45GZsqVQhwHvF1ppQ54qN8EEAibISLqvz5pD_Njf75MNg8QGuDvhJTxkLiiyRUn5hteVVP9YyMjYtmmSxK7YHslFp3WppQWUWAd7HY_sPu2-2Ne0HTBY9zGqqoiqaM1RfzPAqMNKFT4ztLMPoLRAZY1KCFm63MoEtRSvQnKs_xqhYxIH4yUA35zQo';
+    authTokenBad = 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MTgzMiwidXNlcm5hbWUiOiJ1c2VyQmFkIiwiZW1haWwiOiJ1c2VyQmFkQGFtaWRhLmNvbSIsInNjb3BlcyI6WyJhZG1pbiJdLCJpYXQiOjE1MjQwNzA0NzAsImV4cCI6MTUyNDA3NDA3MH0.iO95yeIiKXLgns8nluv1S0jsSqrM9dJsVsB9wvz_mhBgUJWJbOB_GX1s7CETg89z4sILmkLGgEHsIKNgRQ0WDZmGDqFbFJMYzGzar9ujyifuvU7-QMqEWPkf1LYRJE3cZdbzApG--ddk6hPe5i92QEv35wcqbbepbnGcZQbEC6YYeBpErWolA6oBl53sCJArce1pMbH2R7ou6u-Ad4RKaYJkId50UWv-qal6PBVlF665ae0xF9tXtnU0l974kj54o9ufqqw_ysZtLQfjrIULWRTIkuMLh9WG8f_k865DIIgacQyHM-IxMJHNJ6wWm4r1L-AnZLq46qLZVFR1ZSE7rNQHU8R07SqpJbHGcxOw3Ndn9pCohg4TMoFeoMpguhGsxLD-wcA2fhnTt5oEWUjLEFDdt8z99oUZ4geg_xV-qOuZwtJrr-cNRigHCUr53nV4acq61r2TzvavLD_IYG4db7s0zHzKziYQXEG21s9Bav4yoUSvc7gr6FWv-UhukorMus_9OifPJHIQlpOY-07lP1VzMsR-ZOqJXnj9nrNP1mYcwTugyEMyv31pdbk3Se6IRebZwnY5qvLTDSryl1Y0ro2VakDEOxN1mYBhb76x-5c_StCvQiMizkQHV3YiZ4wom6m50X24rH__x9yByi1XvAcdnAvJZxq1WcTuM5t4F4w';
+} else {
+    authToken = config.testToken;
+    auth2Token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6InVzZXIyIiwiZW1haWwiOiJ0ZXN0QHRlc3QuY29tIiwiYWRtaW4iOnRydWV9.IXN3UeBdUHLxVLHEk9a7IuY6DVQcnuA8ykxRR6JdC_k';
+    authTokenBad = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6InVzZXJCYWQiLCJlbWFpbCI6InRlc3RAdGVzdC5jb20iLCJhZG1pbiI6dHJ1ZX0.Bht75P-tmchDXssNb58r8mzwe4rHpNZVNzYHQtzfp5k';
+}
+
 export const baseURL = (version > 0 ? `/api/v${version}` : '/api');
```

Comment:
@rmharrison arghh, I am worried about this breaking tests. Grant (while fixing the messaging service tests changed these vars to be what is below but Jacob had set this file to export the dynamically set authToken, auth2Token, and authTokenBad